### PR TITLE
Issue #922 :Removing some hints in helps-thread

### PR DESCRIPTION
**Pull request by AdityaM2205**
The two obselete hints have been removed entirely from the list
image

### CODE REVIEW

**Review by Zabuzard — DISMISSED** [review]

**Review by Taz03 — DISMISSED** [review]

**Comment by ankitsmt211** [comment]
@AdityaM2205 Are you still upto this? couple of checks failed.

**Comment by AdityaM2205** [comment]
I'm sorry,I tried modifying the dependencies a little,but the tests still don't pass,kindly assign the issue to someone else

**Comment by ankitsmt211** [comment]
> I'm sorry,I tried modifying the dependencies a little,but the tests still don't pass,kindly assign the issue to someone else

No worries, thank you for trying you were almost there. Btw if you get stuck or plan to contribute further feel free to ask questions here or in our server. We'd be happy to help.

**Review by Taz03 — DISMISSED** [review]

**Comment by ankitsmt211** [comment]
@Taz03 Please add the screenshot for new prompts.

File: application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java [hunk 1]
```diff
@@ -111,14 +111,6 @@ public HelpSystemHelper(Config config, Database database, ChatGptService chatGpt
...
     RestAction<Message> sendExplanationMessage(GuildMessageChannel threadChannel) {
-        return mentionGuildSlashCommand(threadChannel.getGuild(), HelpThreadCommand.COMMAND_NAME,
-                HelpThreadCommand.Subcommand.CLOSE.getCommandName())
-                    .flatMap(closeCommandMention -> sendExplanationMessage(threadChannel,
-                            closeCommandMention));
-    }
-
-    private RestAction<Message> sendExplanationMessage(GuildMessageChannel threadChannel,
-            String closeCommandMention) {
         boolean useCodeSyntaxExampleImage = true;
```
Comment (ankitsmt211) [inline application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java:114]: Do we need this syntax example image anymore? I think issue talked about removing this iirc?

**Comment by Taz03** [comment]
note that i can't remove the image from the resources folder coz its not my fork and github doesn't allow me to do that, it'll have to be done in a separate pr

**Review by Taz03 — APPROVED** [review]

File: application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java (hunk 1 above)
Comment (Taz03) [reply application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java]: done

**Comment by Taz03** [comment]
sonarlint is complaining because embedWith have no usage now, should I remove it or should keep it?

**Comment by ankitsmt211** [comment]
> sonarlint is complaining because embedWith have no usage now, should I remove it or should keep it?

i don't see a reason to keep it, the only place it was used is no longer using it.

**Review by Taz03 — APPROVED** [review]

**Review by ankitsmt211 — APPROVED** [review]
seems good.

**Pull request merged**